docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -380,6 +380,12 @@ Required when `actions` are used to send notifications.
 
 * `field_names`: String[] , required
 
+|related_integrations |Object[] a| Fleet integrations the rule depends on. The object has three fields:
+
+* `package` (String, required): Integration package's name EPR uses
+* `integration` (String, optional): Integration's name. It's optional for packages with the only one integration whose name matches package name but required for packages with multiple integrations.
+* `version`: (String, required): Integration (package containing the integration) version constraint in https://semver.org/[semantic versioning] format. For version ranges, you must use tilde or caret syntax. For example, `~1.2.3` is from 1.2.3 to any patch version less than `1.3.0`, and `^1.2.3` is from `1.2.3`` to any minor and patch version less than `2.0.0`.
```

Comment:
> For example, `~1.2.3` is from 1.2.3 to any patch version less than `1.3.0`, and `^1.2.3` is from `1.2.3`` to any minor and patch version less than `2.0.0`.

Do we want to display this example inline with the API field description itself or is there a better way to frame this (like a note or something)? @joepeeples

Reply:
I think it's OK inline as is. I tinkered with a note or new paragraph on my local build, but it looked kinda disruptive.